pkg/cmd/pulumi/new.go

```diff
@@ -340,15 +341,17 @@ options:
 
 	// Install dependencies.
 	if !args.generateOnly {
+		span := opentracing.SpanFromContext(ctx)
 		projinfo := &engine.Projinfo{Proj: proj, Root: root}
-		pwd, _, ctx, err := engine.ProjectInfoContext(projinfo, nil, cmdutil.Diag(), cmdutil.Diag(), false, nil)
+		pwd, _, pluginCtx, err := engine.ProjectInfoContext(
+			projinfo, nil, cmdutil.Diag(), cmdutil.Diag(), false, span)
```

Comment:
Style nit. Feel free to reject this change. I just hate having to search for the closing parenthesis. :) 

```suggestion
		pwd, _, pluginCtx, err := engine.ProjectInfoContext(
			projinfo, 
			nil, 
			cmdutil.Diag(), 
			cmdutil.Diag(), 
			false, 
			span
		)
```